src/tools/idlc/src/plugin.c

```diff
@@ -21,6 +21,10 @@
 static const char sep[] = "/\\";
 static const char lib[] = "";
 static const char ext[] = "dll";
+#define SUBPROCESS_POPEN _popen
+#define SUBPROCESS_PCLOSE _pclose
+#define SUBPROCESS_MODE "rt"
```

Comment:
I'm wondering if text-mode is right. Normally we open files binary so we have actual bytes, because Windows might play tricks. The how-and-what escapes me, but might be good to give that a check if you haven't already.

Reply:
You were right about this one,  in 'r' mode it mimicks UNIX and the windows 'rt' mode does some weird things wrt line endings. The code below was insensitive to the differences, but no need to make things more complex than needed.